doc/api/http.md

```diff
@@ -1392,7 +1389,7 @@ Limits maximum incoming headers count. If set to 0, no limit will be applied.
 added: v14.11.0
 -->
 
-* {number} **Default:** `0`
+* {number} **Default:** `300000`
```

Comment:
We need to add an entry to the changes list in YAML comment for this change

Reply:
Done, thanks for noticing!